apps/aesophia/src/aeso_ast_infer_types.erl

```diff
@@ -166,7 +173,7 @@ infer(Contracts, Options) ->
                     [Alias("oracle", 2), Alias("oracle_query", 2)]
             end,
         create_options(Options),
-        ets:new(type_vars, [set, named_table, public]),
+        ets_new(type_vars, [set, named_table, public]),
```

Comment:
Remove `named_table` and `public`

Reply:
I had the goal to make minimal changes and I didn't know if there was a real reason for having the tables public. I can make a an extra push which cleans up the code a bit internally but then it will have to be approved again.